uber/custom_tags.py

```diff
@@ -1,46 +1,59 @@
 from uber.common import *
 
+
 @register.filter
 def datetime(dt, fmt='11:59pm EST on %A, %b %e'):
-    return ' '.join(dt.astimezone(EVENT_TIMEZONE).strftime(fmt).split())
+    return ' '.join(dt.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split())
+
+from datetime import datetime  # noqa: now that we've registered our filter, re-import the "datetime" class to avoid conflicts
+
 
 @register.filter
 def timestamp(dt):
     from time import mktime
     return str(int(mktime(dt.timetuple())))
 
```

Comment:
Extra lines before function definitions is a PEP8 thing (the most common code formatting standard for Python).